cron/monthly/expiring-cards.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+import '../../server/env';
+
+import logger from '../../server/lib/logger';
+import models from '../../server/models';
+import * as libPayments from '../../server/lib/payments';
+
+// Only run on the first of the month
+const today = new Date();
+const date = today.getDate();
+const month = today.getMonth();
+const year = today.getFullYear();
+
+if (process.env.NODE_ENV === 'production' && date !== 1) {
```

Comment:
Would be great to add the `OFFCYCLE`, similar to other monthly jobs.

Reply:
Notes: OFFCYCLE allows us to fire the script whenever you want, not just the first of the month